target/linux/ramips/dts/OMEGA2.dts

```diff
@@ -0,0 +1,188 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "onion,omega", "mediatek,mt7628an-soc";
+	model = "Onion Omega2";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	bootstrap {
+		compatible = "mediatek,linkit";
+
+		status = "okay";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		system {
+			label = "onion:amber:system";
+			gpios = <&gpio1 12 1>;
+			default-state = "on";
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		wps {
+			label = "reset";
+			gpios = <&gpio1 6 1>;
```

Comment:
The button is actually ACTIVE_HIGH, not ACTIVE_LOW, ie. _<&gpio1 6 0>;_ like I mentioned several times on the forums. The button doesn't work, if you leave it like this.